### PR TITLE
change core lib cosm js

### PR DESCRIPTION
**Pull request by im6h**

### CODE REVIEW

**Comment by ducphamle2** [comment]
This PR follows the [issue](https://github.com/oraichain/orai/issues/8) to upgrade dApps to the new Cosmos SDK version

File: js/cosmjs.js [hunk 1]
```diff
@@ -43,33 +42,39 @@ const queryWasm = async (address, input) => {
...
+    });
+
     try {
-        const rawInputs = [{
-            contractAddr: address,
-            message: Buffer.from(JSON.stringify(handleMsg)),
-        }]
-        const result = await cosmos.execute({ signerOrChild: childKey, rawInputs, gasLimits: 'auto', memo });
-        console.log("result: ", result);
+        const result = await client.execute(wallet, address, Buffer.from(JSON.stringify(handleMsg)), 'auto', memo)
```
Comment (ducphamle2) [inline js/cosmjs.js]: I think the `Buffer.from(JSON.stringify(handleMsg))` looks odd. Can you check it again? I remember that the parameter is an object (JSON.parse), not a Buffer

**Comment by ducphamle2** [comment]
also, do you mind updating the js-server to cosmjs as well?
https://github.com/oraichain/aioracle/blob/master/js-server/submitReportInterval.js#L40

stil have oraiwasm-js left here

**Comment by im6h** [comment]
> stil have oraiwasm-js left here

Let me check it

**Comment by im6h** [comment]
Hi @ducphamle2, I have updated the codebase to `cosmjs`. I need you to verify thoroughly. I will reformat the code after this PR is merged into the master. Many thanks

**Review by ducphamle2 — APPROVED** [review]

**Pull request merged**